tailwind.config.js

```diff
@@ -68,6 +68,9 @@ export default {
         violet: "#b23386",
         gray: "#f4f4f0",
         "dark-gray": "#242423",
+        "light-gray": "#f0f0f0",
+        "gray-300": "#C6C6C6",
+        "gray-500": "#808080",
```

Comment:
@laugardie _a

just wanted to make sure you're fine with adding these new arbitrary colors? feel free to merge afterwards!